### PR TITLE
Add whatsnew update for changes since June 9

### PR DESCRIPTION
**Pull request by shrielenee**
## Purpose of this pull request

This pull request (PR) updates the What's New topic with changes since June 9th.

## Affected DevDocs pages

-  https://devdocs.magento.com/whats-new.html

Reference preview build 1939

## Links to Magento source code

-  ...

### CODE REVIEW

File: src/_data/whats-new.yml [hunk 1]
```diff
@@ -4,8 +4,38 @@ description:
...
+updated: Mon Jun 15 09:55:11 2020
 entries:
+- description: Added two new REST endpoints that add support for the Inventory Management
+    [In-store Pickup](https://devdocs.magento.com/guides/v2.4/rest/modules/inventory/in-store-pickup.html)
+    feature.
+  versions: 2.4.0
+  type: New Topic
+  date: June 12, 2020
+  link: https://github.com/magento/devdocs/pull/7383
+- description: Added the [`pickupLocations` query](https://devdocs/magento.com/guides/v2.4/graphql/queries/pickup-locations.html),
```
Comment (meker12) [inline src/_data/whats-new.yml:16]:
```suggestion
- description: Added the [`pickupLocations` query](https://devdocs.magento.com/guides/v2.4/graphql/queries/pickup-locations.html),
```

**Comment by shrielenee** [comment]
> Looks good. Only a few requests:
> 
> 1. Fix the link pointed by @meker12
> 2. Looks like it is sorted by alphabet, whereas it is supposed to be sorted by date by default.
>    @shrielenee , is it original output of your tool, or you do additional sorting?

I did not do any additional sorting. I bet this is because I removed some duplicates, and perhaps removed the ones that were in chronological order. I'll review the order. Thanks.

**Comment by shrielenee** [comment]
@dshevtsov Please review now.

**Comment by shrielenee** [comment]
running tests

**Pull request merged**